robyn/router.py

```diff
@@ -1,13 +1,11 @@
 import inspect
+import logging
```

Comment:
are you using this?

Reply:
is was already there -- this just got moved here. I will run isort & update this.